lib/foreman_theme_satellite/documentation.rb

```diff
@@ -51,7 +51,7 @@ module Documentation
       'LDAPAuthentication' => "#{ForemanThemeSatellite.documentation_root}/installing_satellite_server_in_a_connected_network_environment/configuring_external_authentication_satellite#Using_LDAP_satellite",
       'Bookmarks' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/searching_and_bookmarking_admin",
       'Policydeploymentoptions' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance#Managing_Compliance_Policies_security-compliance",
-      'CreatingSCAPcontent' => "#{ForemanThemeSatellite.documentation_root}/administering_red_hat_satellite/managing_security_compliance_admin#running-a-security-compliance-scan-on-demand_admin",
+      'CreatingSCAPcontent' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/running-a-security-compliance-scan-on-demand_security-compliance",
```

Comment:
```suggestion
      'CreatingSCAPcontent' => "#{ForemanThemeSatellite.documentation_root}/managing_security_compliance/configuring_scap_contents_security-compliance",
```
If the link source is about creating SCAP content, then perhaps this chapter would be more appropriate as the docs target?

Reply:
Thanks!

I did a bit of archeology, this pointed originally at `/administering_red_hat_satellite/chap-administering-security_compliance_management#sect-Administering-Security_Compliance_Management_with_OpenSCAP-Configuring_SCAP_Content`, which was e.g. the structure in 6.10:
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/chap-administering-security_compliance_management#sect-Administering-Security_Compliance_Management_with_OpenSCAP-Configuring_SCAP_Content
This chapter is called "7.2. Configuring SCAP Content" and talks about importing the Puppet modules needed for operation, followed by the actual SCAP content (e.g. from `scap-security-guide`)

And that's exactly where your suggestion is now leading us again. Updating.